frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -67,7 +78,7 @@ const UsersTable = ({
               <tbody>
                 {Array.isArray(users) &&
                   users.length > 0 &&
-                  users.map((user) => (
+                  sortUsersByFirstName(users).map((user) => (
```

Comment:
Changes should be applied to the backend and not, the front end.
API call should return results in an alphabetically sorted format.

Reply:
Ok I'll do that give me some time

Reply:
Made all the necessary changes! 😊